src/backend/stream.rs

```diff
@@ -406,7 +465,7 @@ impl<'ctx> StreamOps for PulseStream<'ctx> {
         self.shutdown = false;
         self.cork(CorkState::uncork() | CorkState::notify());
 
-        if self.output_stream.is_some() && self.input_stream.is_none() {
+        if self.output_stream.is_some() {
             /* On output only case need to manually call user cb once in order to make
```

Comment:
`/* On output only case need to ....` not output only anymore? If so, we should update the comment here.

Reply:
Fixed, thanks.